src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -7571,11 +7568,12 @@ void CalcHeatBalanceInsideSurf(EnergyPlusData &state,
         state.dataHeatBalSurfMgr->calcHeatBalInsideSurEnvrnFlag = true;
     }
 
+    sumSurfQdotRadHVAC(state);
```

Comment:
This is the main fix for the defect. A new function to gather all of the radiant impacts on the pertinent surfaces. One call here no matter the surface type or the heat transfer algorithm.

Reply:
:+1: